src/pymor/core/config.py

```diff
@@ -98,6 +98,8 @@ def is_nbconvert():
 _PACKAGES = {
     'CYTHON': lambda: import_module('cython').__version__,
     'DEALII': lambda: import_module('pydealii'),
+    'DOCOPT': lambda: import_module('docopt').__version__,
```

Comment:
This isn't used in the PR atm, right? Most(?) demos use `typer` instead of `docopt`.

Reply:
I think all demos use typer now. docopt should not be used.

Reply:
this was a merge error, not present in #1383